opendevin/runtime/plugins/swe_agent_commands/cursors_edit_linting.sh

```diff
@@ -35,8 +35,8 @@ edit() {
     local new_lines=("${lines[@]:0:$start_line}" "${replacement[@]}" "${lines[@]:$((end_line))}")
     # Write the new stuff directly back into the original file
     printf "%s\n" "${new_lines[@]}" >| "$CURRENT_FILE"
-    # Run linter
-    if [[ $CURRENT_FILE == *.py ]]; then
+    # Run linter if enabled
+    if [[ $CURRENT_FILE == *.py && -n "$LINT_ENABLED" ]]; then
```

Comment:
let's make the naming a bit clearer

```suggestion
    if [[ $CURRENT_FILE == *.py && -n "$RUN_LINT_AFTER_EDIT" ]]; then
```